emails/views.py

```diff
@@ -609,14 +609,16 @@ def _handle_bounce(message_json):
     bounce = message_json.get('bounce')
     bounced_recipients = bounce.get('bouncedRecipients')
     for recipient in bounced_recipients:
-        recipient_address = recipient.pop('emailAddress', None)
+        recipient_address = recipient.get('emailAddress', None)
+        if recipient_address is None:
+            continue
         recipient_address = parseaddr(recipient_address)[1]
         recipient_domain = recipient_address.split('@')[1]
         capture_message(
             f'bounced recipient domain: {recipient_domain}, {recipient}'
         )
         try:
-            user = User.objects.get(email=recipient.get('emailAddress'))
+            user = User.objects.get(email=recipient_address)
```

Comment:
We have a same problem with `recipient_address` being only the domain portion since in line 615:
```
recipient_address = parseaddr(recipient_address)[1]
```

Reply:
[The return value of `parseaddr` is a tuple](https://docs.python.org/3.8/library/email.utils.html#email.utils.parseaddr) where the `0`th item is the "real name" and the `1`st item is the email address.

I confirmed by triggering a bounce to my local app. When SES/SNS sent Relay the bounce message, the `emailAddress` field comes in like:

```
ipdb> recipient.get('emailAddress', None)
'bounce@simulator.amazonses.com'
ipdb> recipient_address
'bounce@simulator.amazonses.com'
ipdb> parseaddr(recipient_address)
('', 'bounce@simulator.amazonses.com')
ipdb> parseaddr(recipient_address)[1]
'bounce@simulator.amazonses.com'
```

Reply:
oh my bad. I should have looked into that more. Thanks for clarifying that.